### PR TITLE
Rework the Rescue Hardsuit

### PR DESCRIPTION
**Pull request by Arokha**
Module 1: Maneuvering Jets - Jetpack, fwoosh.
Module 2: Sprinter - Gotta go fast!
Module 3: PATS Module - Opens doors you have access to 3 steps ahead of you, and has an (expensive) option to override an airlock you're facing, though this notifies command and sec of you doing this over radio.
Module 4: Micro Pharmacy - Contains 10u inaprovaline, tricordrazine, tramadol, and dexplus, each, and can have regeneration enabled to slowly regenerate these four drugs in exchange for energy.

(Also fixed a thing with the resleeving computer and disk while I was in there)

### CODE REVIEW

File: code/modules/clothing/spacesuits/rig/modules/utility_vr.dm [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		return 0
+
+	var/obj/machinery/door/airlock/A = locate(/obj/machinery/door/airlock) in get_step(H,H.dir)
+
+	//Okay, we either found an airlock or we're about to give up.
+	if(!A || !A.density || !A.can_open() || !..())
+		to_chat(H,"<span class='warning'>Unable to comply! Energy too low, or not facing a working airlock!</span>")
+		return 0
+
+	H.visible_message("<span class='warning'>[H] begins overriding the airlock!","<span class='notice'>You begin overriding the airlock!</span>")
```
Comment (Yoshax) [inline code/modules/clothing/spacesuits/rig/modules/utility_vr.dm]: Unclosed span class.

**Comment by SpadesNeil** [comment]
I got dis

File: code/modules/clothing/spacesuits/rig/modules/utility_vr.dm (hunk 1 above)
Comment (SpadesNeil) [inline code/modules/clothing/spacesuits/rig/modules/utility_vr.dm]: I got this. :U

**Comment by Arokha** [comment]
Thanks!

**Pull request merged**